webpack.config.js

```diff
@@ -14,10 +14,18 @@ var config = {
     libraryTarget: 'umd'
   },
   plugins : [
+    {
+      apply: function apply(compiler) {
+        compiler.parser.plugin('expression global', function expressionGlobalPlugin() {
+          this.state.module.addVariable('global', "(function() { return this; }()) || Function('return this')()")
```

Comment:
You stated that double compiling is an edge case. One that webpack clearly doesn't support. This is going above and beyond to support them in the primary case. If they are double compiling and using their code in extensions then yes it could be an issue.

Lodash's UMD is erm robust because folks do some crazy stuff and we have to support it. I think your case is fine though.